var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -124,6 +127,9 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
     depends_on('binutils~libiberty', when='+binutils', type=('build', 'link', 'run'))
     depends_on('zip', type='build', when='languages=java')
     depends_on('cuda@:10', when='+nvptx')
+    #depends_on('llvm-amdgpu', when='+amdgcn') # not binary compatible with llvm9 fmt.
+    # FIXME: there's no way to specify this in spack, it's a variant, not a dep
+    #depends_on('languages=lto', when='+amdgcn')
```

Comment:
Note: The addition of link time optimization is a suggestion from https://gcc.gnu.org/wiki/Offloading, not a requirement.